src/components/App/PuzzleGenerate/index.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import styles from './PuzzleGenerate.module.scss';
+import { Button, Input } from '../../Global';
+import { createPuzzleInstance, getAllPuzzles } from '../../../services';
+
+const PuzzleGenerate = () => {
+  const [puzzleList, setPuzzleList] = useState([]);
+  const [hint, setHint] = useState('');
+  const [latitude, setLatitude] = useState('');
```

Comment:
why are we tracking latitude, longitude, and id as strings in the state? Aren't they numbers?

Reply:
they are numbers